### DISCUSSION TITLE
Invalid 'main' field in nuxt-theme/package.json

### DISCUSSION THREAD

**Discussion by sbkjarmul** [vuestorefront/prestashop#28] [Q&A]
Hello,

When I start nuxt server with `yarn dev` I get the following error:
<img width="1138" alt="image" src="https://user-images.githubusercontent.com/37276167/165166350-b6886c03-a0b1-48ae-bfe0-e524a228e234.png">

I haven't changed that file. How to solve it? What should be set as main there?
<img width="535" alt="image" src="https://user-images.githubusercontent.com/37276167/165166472-3c6a7e34-bfb7-4f7d-81f3-76e86fbb05d9.png">

**Comment by samberrry** [comment]
Hello,
it's just warning, you can ignore it.

**Comment by samberrry** [comment]
same warning with boilerplate. Reported:
https://github.com/vuestorefront/ecommerce-integration-boilerplate/issues/56
